.github/stale_labeler.yml

```
@@ -0,0 +1,6 @@
+version: 1
+appendOnly: false
+labels:
+- label: "Status: Stale"
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 1

(indentation)